bundles/org.openhab.binding.webthing/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="webthing"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="generic">
+		<label>WebThing Binding Thing</label>
+		<description>Sample thing for WebThing Binding</description>
+
+		<config-description>
+			<parameter name="webThingURI" type="text" required="true">
+				<label>webThingURI</label>
+				<description>This is the url of the webthing </description>
```

Comment:
Can you fix the capitalization?

Reply:
Do you mean the description?. Camel case is OK for parameter name and label, or iisn't it?

Reply:
Yes, the description. But the label should be human readable.